templates/patterns/page_header/page_header.ui_patterns.yml

```diff
@@ -1,6 +1,13 @@
 page_header:
   label: "Page header"
   description: "Default implementation of the page header, contains the page title, breadcrumbs and additional page details."
+  variants:
+    default:
+      label: "Page header: default"
+      description: "Default page header contains the page title, breadcrumbs and additional page details."
+    background_image:
+      label: "Page header: with background image"
+      description: "Page header contains the page title, breadcrumbs and additional page detail with background image."
```

Comment:
This is a bit odd, I understand the idea of a variant, but then, even if the variant is `background_image`, when we don't have an image or we are not using a core branding, then the `background_image` variant will look like the `default`.

Should we simply add the `background_image` field to this pattern, specifying in its description that it will only be shown if ECL branding is set to `core` and an email is actually set?

Reply:
Rework is done. The previous implementation was based on the old convention that we have to keep the pattern structure unchanged and in the case of a new field - create a new variant.